src/lib/storage/fake_device_factory.rb

```diff
@@ -162,6 +162,9 @@ def create_disk(_parent, args)
         @disks << name
         disk = ::Storage::Disk.create(@devicegraph, name)
         disk.size_k = size.size_k
+        # range (number of partitions that the kernel can handle) used to be
+        # 16 for scsi and 64 for ide. Now it's 256 for most of them.
+        disk.range = args["range"] || 256
```

Comment:
I prefer having 256 as constant as it is more descriptive, but it is used only here and with comment, it is fine for me.